test/functional/specs/desktop/C8119.js

```diff
@@ -20,9 +20,28 @@ test.meta({
   TEST_RUN: "Regression"
 });
 
-test("Test C8119: Load page with link. Click link. Verify no request sent.", async () => {
+test("Test C8119: Load page with link. Click link. Verify no event sent.", async () => {
+  const logger = createConsoleLogger(t, "log");
+  const testConfig = {
+    clickCollectionEnabled: false,
+    onBeforeEventSend(options) {
+      try {
+        // eslint-disable-next-line no-console
+        console.log(options.xdm.web.webInteraction.URL);
+        // eslint-disable-next-line no-empty
+      } catch (e) {}
+    }
+  };
+  Object.assign(testConfig, baseConfig);
+  await configureAlloyInstance("alloy", testConfig);
+  await addAnchorToBody({
+    text: "Test Link",
+    attributes: {
+      href: "blank.html",
+      id: "alloy-link-test"
+    }
+  });
   await t.click(Selector("#alloy-link-test"));
-  await responseStatus(networkLogger.edgeEndpointLogs.requests, 200);
-  const gatewayRequest = networkLogger.edgeEndpointLogs.requests[0];
-  await t.expect(gatewayRequest).eql(undefined);
+  const newMessages = await logger.getNewMessages();
```

Comment:
There is a similar race condition here.  Shouldn't we wait until the page redirected before checking to see if the event was sent?